testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -525,19 +530,34 @@ impl ClusterSwarmKube {
             .allocate_node(&pod_name)
             .await
             .map_err(|e| format_err!("Failed to allocate node: {}", e))?;
-        debug!("Configuring fluent-bit for pod {} on {:?}", pod_name, node);
+        debug!(
+            "Configuring fluent-bit, genesis and config for pod {} on {}",
+            pod_name, node.name
+        );
         match &instance_config.application_config {
-            Validator(_) => {
-                self.config_fluentbit("validator-events", pod_name.as_str(), &node.name)
-                    .await?
+            Validator(validator_config) => {
+                self.config_fluentbit("validator-events", &pod_name, &node.name)
+                    .await?;
+                if validator_config.enable_mgmt_tool {
```

Comment:
if it does not have order have to after config_fluentbit, u may can move this if condition outside. Otherwise I think its good.

Reply:
This if is only used to gate the case LSR/Vault is enabled or not. So I think it should be deleted.